src/libs/actions/PersonalDetails.js

```diff
@@ -50,6 +50,23 @@ function getDisplayName(login, personalDetail) {
     return fullName || userLogin;
 }
 
+/**
+ *
+ * @param {Number} accountID
+ * @param {String} [defaultDisplayName] display name to use if user details don't exist in Onyx or if
+ *                                      found details don't include the user's displayName or login
+ * @returns {String}
+ */
+function getDisplayNameByAccountID(accountID, defaultDisplayName = '') {
```

Comment:
Why we are writing a new function instead of overwriting the old one `getDisplayName`